source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,253 @@
+:ref:`csharp-retrieve-additional-information`.
+
+Find Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To find multiple documents in a collection, pass a query filter to the ``Find()``
+method that specifies the criteria of the documents you want to retrieve.
+
+You can use a **cursor** to iterate over the documents returned by the ``Find()``
+method. A cursor is a mechanism that allows an application to iterate over database 
+results while holding only a subset of them in memory at a given time. Cursors
+are useful when your ``Find()`` method returns a large amount of documents.
+
+To use a cursor to iterate over the documents, pass a 
+query filter to the ``Find()`` method that specifies the criteria of the documents 
+you want to find, then chain the ``ToCursor()`` or ``ToCursorAsync()`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-cursor-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToCursorAsync();
+
+   .. tab:: Synchronous
+      :tabid: find-cursor-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = _restaurantsCollection.Find(filter).ToCursor();
+
+If you are returning a small number of documents, or need your results returned
+as a ``List`` object, use the ``ToList()`` or ``ToListAsync()`` methods.
+
+To find multiple documents in a collection and hold them in memory as a list, pass a query filter 
+to the ``Find()`` method that specifies the criteria of the documents you want 
+to find, then chain the ``ToList()`` or ``ToListAsync()`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-list-async
+
+      .. code-block:: csharp
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToList();
+
+   .. tab:: Synchronous
+      :tabid: find-list-sync
+
+      .. code-block:: csharp
+
+         var restaurants = _restaurantsCollection.Find(filter).ToListAsync();
+
+To see a full example of using the ``Find()`` method to find multiple documents,
+see :ref:`csharp-retrieve-additional-information`.
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+To find all documents in a collection, pass an empty
+``BsonDocument`` as a query filter. Then chain the ``ToList()`` or ``ToListAsync()``
+method to return the documents as a ``List`` object, or the ``ToCursor()`` or ``ToCursorAsync()``
+method to return the documents as a cursor.
+
+To see a fully runnable example of using the ``Find()`` method to find all documents, see 
+:ref:`csharp-retrieve-additional-information`.
+
+Modify Find Behavior
+~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of the ``Find()`` method by passing
+a ``FindOptions`` object.
+
+You can configure the commonly used options with the following methods:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``BatchSize`` 
```

Comment:
Question: Any reason this did not receive method formatting `()`?

Reply:
Adding `()` made it seem like you needed to call it like a method if you wanted to use it, so I left it since you pass it in an options object when using it.